modules/gdscript/gdscript.cpp

```diff
@@ -603,6 +603,24 @@ Error GDScript::reload(bool p_keep_state) {
 
 	valid = true;
 
+	GDScript *top = this;
+	while (top) {
+
+		Map<StringName, GDScriptFunction *>::Element *E = top->member_functions.find("can_instance");
+		if (E) {
+
+			if (!E->get()->is_static()) {
+				WARN_PRINT(String("Can't call non-static function: 'can_instance' in script.").utf8().get_data());
```

Comment:
Just FYI, you could use `WARN_PRINTS("Can't call non-static function: 'can_instance' in script.");` directly.